libbeat/docs/outputconfig.asciidoc

```diff
@@ -1186,3 +1211,30 @@ The following elliptic curve types are available:
 * P-256
 * P-384
 * P-521
+
+[[configuration-output-codec]]
+=== Output Codec Configuration
+
+Outputs that do not require a specific output encoding do support to change the encoding using the `codec` configuration. The codec configured must be one of `json` or `format`. By default the `json` codec is used.
```

Comment:
It might be more clear to say: "For outputs that do not require a specific encoding, you can change the  encoding by using the `codec` configuration. You can specify either the `json` or `format` codec. By default the `json` codec is used.